net-misc/icaclient/icaclient-22.11.0.19.ebuild

```diff
@@ -0,0 +1,260 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop wrapper xdg-utils
+
+DESCRIPTION="ICA Client for Citrix Presentation servers"
+HOMEPAGE="https://www.citrix.com/"
+SRC_URI="amd64? ( linuxx64-${PV}.tar.gz )
+	x86? ( linuxx86-${PV}.tar.gz )"
+
+LICENSE="icaclient"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="l10n_de l10n_es l10n_fr l10n_ja l10n_zh-CN"
+RESTRICT="mirror strip fetch"
+
+ICAROOT="/opt/Citrix/ICAClient"
+
+QA_PREBUILT="${ICAROOT#/}/*"
+
+# we have binaries for two conflicting kerberos implementations
+# https://bugs.gentoo.org/792090
+# https://bugs.gentoo.org/775995
+REQUIRES_EXCLUDE="
+	libgssapi.so.3
+	libgssapi_krb5.so.2 libkrb5.so.3
+"
+# we have binaries which wouls still support gstreamer:0.10
+REQUIRES_EXCLUDE="${REQUIRES_EXCLUDE}
+	libgstapp-0.10.so.0
+	libgstbase-0.10.so.0
+	libgstinterfaces-0.10.so.0
+	libgstpbutils-0.10.so.0
+	libgstreamer-0.10.so.0
+"
+RDEPEND="
+	|| (
+		>=app-accessibility/at-spi2-core-2.46.0:2
+		dev-libs/atk
+	)
```

Comment:
This should be unnecessary now, but no harm keeping it for now.

Reply:
ok, i have to admit i did not follow why how this came in, and i received a strange email from a bot when it was merged ... but so far ignored that